keyboards/dichotomy/keymaps/default/keymap.c

```diff
@@ -0,0 +1,517 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
+#include QMK_KEYBOARD_H
+#include "report.h"
```

Comment:
I see both of these in a number of places.  It may be best to add these to the keyboard's H file, actually. 

That way, they don't need to be explicitly defined everywhere.